src/scenic/simulators/Gazebo/README.md

```diff
@@ -0,0 +1,76 @@
+﻿# Scenic-Gazebo Interface
+This repo contains the code to the Scenic-Gazebo interface.
+## Requirements & Installation
+
+ - Scenic 3 is required, as well as Python 3.8+, ROS Noetic, and Ubuntu 20.04.
+ - The Ubuntu 20.04 requirement is due to ROS Noetic officicially supporting it. It should not be strictly required if you can get ROS Noetic running with Gazebo on your machine. Though this interface is only tested on Ubuntu 20.04.
+ - Other than the official installation, ROS Noetic can also be installed with [RoboStack](https://robostack.github.io/index.html), which can be helpful if you are on MacOS or Windows.
+ - Before you install, it might be good to consider installing Scenic outside any virtual environment for Gazebo/ROS purposes since Gazebo/ROS utilizes some Ubuntu native packages that is difficult to get from conda/pip
+ - When installing Scenic, please do the "Repository" install outlined in this page: https://scenic-lang.readthedocs.io/en/latest/quickstart.html#installation 
+ - In terms of using ROS and Gazebo, it should be fine to control your robot without ROS. However, Gazebo needs to be started with ROS.
+
+## Instructions for Use
+
+ 1. Before running Scenic, start the Gazebo simulator with ROS, launch and bringup any ROS stack you need, and unpause the
+    simulation. 
+ 2. Open a separate terminal, run `scenic <YOUR PROGRAM>.scenic --simulate`(see https://scenic-lang.readthedocs.io/en/latest/options.html for more
+    command-line options). Make sure you sourced the corresponding `.bash` or `.zsh` needed to run your simulaton in this terminal, too!.
```

Comment:
By sourcing to run the simulation do you mean `source /opt/ros/noetic/setup.bash`? If so I think we should make explicit mention of that here!